components/Carousel/CarouselItem.js

```diff
@@ -1,24 +1,74 @@
 import React from 'react';
 import { ParallaxImage } from 'react-native-snap-carousel';
 import { View, Text, Pressable, SafeAreaView } from 'react-native';
-import styles from './styles';
+// import styles from './styles';
+import GoodVibeModal from '../Navigation/GoodVibeModal';
+import { Dimensions, StyleSheet, Platform } from 'react-native';
 
 function CarouselItem({ item, index }, parallaxProps) {
   return (
-    <Pressable onPress={() => alert('Image description:' + item.description)}> 
-      <SafeAreaView style={styles.item}>
-        <ParallaxImage
```

Comment:
We might need - to add parallax back in for modal images- this allows elements to be shown inside a carousel in react. It might be the same in RN